mlflow/utils/databricks_utils.py

```diff
@@ -818,7 +825,7 @@ def _init_databricks_cli_config_provider(entry_point):
 
     dbr_major_minor_version = get_databricks_runtime_major_minor_version()
 
-    if dbr_major_minor_version >= (13, 2):
+    if dbr_major_minor_version[0] == "client" or dbr_major_minor_version >= (13, 2):
```

Comment:
nit: Let's add a comment that the definition of databricksCli.py in client-branch-1.0 is the same as in 15.0